src/state/multicall/retry.ts

```diff
@@ -47,6 +47,7 @@ export function retry<T>(
         }
         break
       } catch (error) {
+        console.error(error)
```

Comment:
I think this might be a bit too much to console.error. Also probably not gonna be useful really :)

Reply:
Since we won't have wallet injection anymore, without this we will not see the error I think